nemoguardrails/library/autoalign/README.md

```diff
@@ -562,7 +562,7 @@ rails:
   config:
     autoalign:
       parameters:
-        fact_check_endpoint: "https://nvidia.autoalign.ai/factcheck"
+        fact_check_endpoint: "<AUTOALIGN_ENDPOINT>"
```

Comment:
or, at least should only apply to the "nvidia.autoalign.ai" portion of the url, the endpoints are pre-defined constants

Reply:
it should look like this? `https://<AUTOALIGN_ENDPOINT>.autoalign.ai/factcheck`

Reply:
or https://<AUTOALIGN_ENDPOINT>/factcheck , this is probably better actually, since our saas platform uses `https://app.autoalign.ai/service/guardrail`